pyrobuf/protobuf/templates/proto_pxd.tmpl

```diff
@@ -52,7 +52,7 @@ cdef class {{ name }}:
 {%- macro enumdef(name, enum) %}
 cdef enum {{ name }}:
     {%- for field in enum.fields %}
-    {{ field.name }} = {{ field.value }}
+    _{{ field.name }} = {{ field.value }}
```

Comment:
Suppose you had the following file called `my_enum.proto`:

```
enum Status {
    OPEN = 1;
    CLOSE = 2;
}
```

In Google's protobuf, the `my_enum_pb2` module would have two top-level attributes called `OPEN` and `CLOSE` with the values 1 and 2 respectively. To replicate this behaviour, I've modified pyrobuf's templates to produce the following the in `pxd` file:

```
cdef enum Status:
    _OPEN = 1
    _CLOSE = 2
```

and the following in the `.pyx` file:

```
OPEN = _OPEN
CLOSE = _CLOSE
```

Since the top level attributes of the module have to have the same names as the enum fields, I needed to rename the enum fields (by prepending an underscore) to avoid name collisions.